src/Model/collections.ts

```diff
@@ -75,6 +76,21 @@ declare module './Model' {
     getOrCreateCollection(name: string): Collection;
     getOrCreateDoc(collectionName: string, id: string, data: any);
 
+    /**
+     * Get a value that may be undefined but ensure value is returned
+     *
+     * @param subpath
+     * @param defaultValue value to return if no value at subpath
+     */
+    getOrDefault<S>(subpath: Path, defaultValue: S): ReadonlyDeep<S>;
+
+    /**
+     * Get a value and throw error if undefined
```

Comment:
Bit of rewording to match the existing get methods' docs, and to clarify what conditions will cause and error

> Gets the value located at this model's path or a relative subpath.
> 
> If no value exists at the path, or the value is nullish (null or undefined), this will throw an error.